### DISCUSSION TITLE
replacing-google-with-searxng-as-the-default-in-chrome/

### DISCUSSION THREAD

**Discussion by giscus** [ironicbadger/blog.ktz.me#234] [Announcements]
# replacing-google-with-searxng-as-the-default-in-chrome/

No tracking. No ads. Search as it was.
Now when you enter a search query in the Chrome address bar it will route through your SearXNG instance automatically.

https://blog.ktz.me/replacing-google-with-searxng-as-the-default-in-chrome/

**Comment by JonCastaway** [comment]
This is great. I have been testing SearXNG on Umbrel with Tailscale Magic DNS and an NPM reverse proxy for SSL.

I have an issue with my primarily use of Vanadium browser, I can not fathom out a way to change the search engine. It does not show as a recent search site in the list. I will to do some yak shaving and get it working as my default search engine

**Comment by Wayne-Harbart** [comment]
This has been fantastic until this past weekend when suddenly I only timeout.  Guess Ill have to sort out why thats happening.

**Comment by Glen-Sumner** [comment]
Fab Alex just need to add shadowsocks to my tailnet now too any ideas?

**Comment by mimnets** [comment]
I found this very user friendly and very helpful. though I have had installed SearXNG and hosted on my cloud server with docker. But not with Proxmox, today I learnt how can I use it with Proxmox. thanks so much for your guide

**Comment by zpthree** [comment]
Thanks for this! Super helpful!
